msr/lotus/steady_state/run_ns_initial_res.i

```diff
@@ -0,0 +1,1092 @@
+    block_id = '5'
+    block_name = 'mixing-plate'
+  []
+  [delete_reactor_caps_bdry]
+    type = BoundaryDeletionGenerator
+    input = mixing_plate
+    boundary_names = 'wall-reactor-caps'
+  []
+  [mix_plate_downstream]
+    type = ParsedGenerateSideset
+    input = 'delete_reactor_caps_bdry'
+    new_sideset_name = 'mixing-plate-downstream'
+    included_subdomains = ' mixing-plate'
+    combinatorial_geometry = ' (x > -0.23 & x < -0.2) & (y*y + z*z<0.12*0.12) '
+  []
+  [add-cooled-wall-pipes-outshield]
+    type = ParsedGenerateSideset
+    input = mix_plate_downstream
+    combinatorial_geometry = 'y>1.'
+    included_boundaries = wall-pipe
+    new_sideset_name = 'heat-loss-section-outshield'
+  []
+  [add-cooled-wall-pipes-inshield]
+    type = ParsedGenerateSideset
+    input = add-cooled-wall-pipes-outshield
+    combinatorial_geometry = 'y<1.'
+    included_boundaries = wall-pipe
+    new_sideset_name = 'heat-loss-section-inshield'
+  []
+  [heated-reflector_walls]
+    type = ParsedGenerateSideset
+    input = 'add-cooled-wall-pipes-inshield'
+    new_sideset_name = 'heated-reflector-walls'
+    included_subdomains = 'reflector'
+    combinatorial_geometry = ' (x < 1.151 & y < -0.7749) | (x < 1.151 & y > 0.7749) |
+                               (x < 1.151 & z < -0.5749) | (x < 1.151 & z > 0.5749)'
+  []
+  [non-heated-reflector_walls]
+    type = ParsedGenerateSideset
+    input = 'heated-reflector_walls'
+    new_sideset_name = 'non-heated-reflector-walls'
+    included_subdomains = 'reflector'
+    combinatorial_geometry = ' (x > 1.149) | (x < -0.24)'
+  []
+  [reactor_out_bdry]
+    type = ParsedGenerateSideset
+    input = 'non-heated-reflector_walls'
+    new_sideset_name = 'reactor_out'
+    combinatorial_geometry = '(y > 775) | ((y <= 775) & (x > 1150 | x < -250))'
+    include_only_external_sides = true
+  []
+[]
+
+################################################################################
+# EQUATIONS: VARIABLES, KERNELS & BCS
+################################################################################
+
+[UserObjects]
+  [pins_rhie_chow_interpolator]
+    type = PINSFVRhieChowInterpolator
+    block = 'reactor pipe pump mixing-plate'
+  []
+[]
+
+[Variables]
+  [pressure]
+    type = INSFVPressureVariable
+    initial_condition = 0.1
+    block = 'reactor pipe pump mixing-plate'
+  []
+  [superficial_vel_x]
+    type = PINSFVSuperficialVelocityVariable
+    initial_condition = 1e-6
+    block = 'reactor pipe pump mixing-plate'
+  []
+  [superficial_vel_y]
+    type = PINSFVSuperficialVelocityVariable
+    initial_condition = 1e-6
+    block = 'reactor pipe pump mixing-plate'
+  []
+  [superficial_vel_z]
+    type = PINSFVSuperficialVelocityVariable
+    initial_condition = 1e-6
+    block = 'reactor pipe pump mixing-plate'
+  []
+  [lambda]
+    family = SCALAR
+    order = FIRST
+    initial_condition = 1e-6
+    block = 'reactor pipe pump mixing-plate'
+  []
```

Comment:
Thanks. Our whole SQA process for MCRE engineering is already based in this lambda Lagrange multiplier. The main reason is that we froze blue_crab before this UO was introduced. Since this need to be consistent with what other members in the MCRE team are seeing, we'll have to keep this normalization for now. We will update in the future